### DISCUSSION TITLE
Vertex colours issue

### DISCUSSION THREAD

**Discussion by Killing-J0ke** [BadDogSkyrim/PyNifly#216] [Q&A]
Hi, I'm on Blender 4.0 and I'm desperately trying to export nifs for Skyrim SSE. Vertex colour flag is always being removed from the trishape desc (not the shader flag) thus causing black models. It's impossible to fix manually (too many nodes) and SNIFF can only batch vertex shaders so I'm a bit lost. Did I miss something when exporting my model ?
Also, what's the method to install the addon when it fails, I tried to install it on a more recent Blender version but installing it the usual way simply doesn't work. Addon is installed but nowhere to be found and thus can't be activated.

**Comment by BadDogSkyrim** [comment]
Had a look and can't reproduce the problem.

1. Always best to point me at a specific nif that shows the problem, if you can. I tested farmbench01 and it imports and exports with the color flag set.
2. I don't know why installing the addon should fail. What version of pynifly, and what version of blender? What error do you get? How do you know the addon was installed if it's nowhere to be found?

**Comment by Killing-J0ke** [comment]
Sorry about that, let me share links for both meshes ([original one](https://drive.google.com/file/d/1TSPs699lC6Y6PqnXDqiY6J495PDBilUZ/view?usp=sharing) vs [tweaked and exported one](https://drive.google.com/file/d/1uzfIwWfnpQgzv09eSw_-pzdifjadbBFu/view?usp=sharing))
Trishapes I partitioned can lose their vertex colours desc flags, in that mesh it's only a few nodes but in other circumstances it can be almost all of them which is impossible to fix manually, way too tedious at least. ^ ^"

(I'm using the io_scene_nifly 14.5.2 addon on Blender 4.0 btw. I tried to install it on blander 4.1 but it didn't work)
